recipes/metric_learn/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2015 CJ Carey and Yuan Tang
```

Comment:
If you are manually packaging the license file, please put in an issue or better yet, a pull request to the upstream repository to include this in the source distribution, as is required by the terms of the license. If you put in an issue or PR, put a comment in the `meta.yaml` above the `license_file` line linking to that issue as a reminder to check if the manually packaged one can be deleted in future updates.

Reply:
Thanks, done https://github.com/scikit-learn-contrib/metric-learn/pull/283